styles/scss/components/menu/menu-overlay.scss

```diff
@@ -0,0 +1,65 @@
+.video-react-options-close {
+  position: absolute;
+  top: 0.4em;
+  right: 0.5em;
+  color: white;
+  font-size: 2.5em;
+  z-index: 2;
+
+  &:hover {
+    cursor: pointer;
+    opacity: 0.8;
+  }
+}
+
+.video-react-options-overlay {
+  position: absolute;
+  top: 0;
+  right: 0;
+  height: 100%;
+  width: 325px;
+  background: rgba(#000000, 0.8);
+  z-index: 1;
```

Comment:
Great catch. I'm not sure why that was the case at the time I took this screenshot, but since your comment was made, this has been fixed.
<img width="628" alt="Screen Shot 2020-03-30 at 12 29 51 PM" src="https://user-images.githubusercontent.com/60157417/77937434-53796100-7282-11ea-8f70-67852d8d0c80.png">